### PR TITLE
Use WGET instead of CURL

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by angristan** [comment]
Why?

**Comment by ghost** [comment]
```
Usage
First, download the script and make it executable:

wget https://raw.githubusercontent.com/Angristan/Local-DNS-resolver/master/unbound-install.sh
chmod +x unbound-install.sh
Then run it as root:

sudo ./unbound-install.sh
```

in the README.MD its wget so lets just use that.

and

on ubuntu and debian you need to install curl and wget comes with the system.

File: unbound-install.sh [hunk 1]
```diff
@@ -83,7 +83,7 @@ if [[ "$OS" = "arch" ]]; then
   pacman -Syu unbound
 
   # Get root servers list
-  curl -o /etc/unbound/root.hints https://www.internic.net/domain/named.cache
+	wget -O /etc/unbound/root.hints https://www.internic.net/domain/named.cache
```
Comment (angristan) [inline unbound-install.sh:86]: Please fix the indentation

Comment (ghost) [reply unbound-install.sh:86]: deleted this branch a while ago, sorry cant make anymore changes to this file.

if you want i can make a new pull.

Comment (angristan) [reply unbound-install.sh:86]: As you want :)

**Pull request closed without merging**